app/controllers/publishers_controller.rb

```diff
@@ -219,7 +212,6 @@ def create_auth_token
       return redirect_to new_auth_token_publishers_path
     end
 
-    @publisher = Publisher.new(publisher_create_auth_token_params)
     @should_throttle = should_throttle_create_auth_token? || params[:captcha].present?
```

Comment:
Should we move `|| params[:captcha].present?` into `should_throttle_create_auth_token?` here and throughout?